### PR TITLE
Replaced docutils link by PyPI reference.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: docs/ref/contrib/admin/admindocs.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ the following:
   your ``urlpatterns``. Make sure it's included *before* the
   ``'admin/'`` entry, so that requests to ``/admin/doc/`` don't get
   handled by the latter entry.
-* Install the docutils Python module (https://docutils.sourceforge.io/).
+* Install the :pypi:`docutils` Python module.
```
Comment (timgraham) [inline docs/ref/contrib/admin/admindocs.txt:26]: module -> package?

Comment (claudep) [reply docs/ref/contrib/admin/admindocs.txt]: Sure, done.

**Comment by felixxm** [comment]
> Maybe, I'm not sure. Or just the second one?

The 2nd sounds good :+1:

**Comment by claudep** [comment]
Thanks @felixxm for finishing the work, too much to do these days...

**Pull request merged**